src/main/webapp/js/common.js

```diff
@@ -83,6 +83,12 @@ var FEEDBACK_RESPONSE_ID = 'responseid';
 var FEEDBACK_RESPONSE_COMMENT_ID = 'responsecommentid';
 var FEEDBACK_RESPONSE_COMMENT_TEXT = 'responsecommenttext';
 
+// Status message status
+var STATUS_SUCCESS = "success";
+var STATUS_INFO = "info";
+var STATUS_WARNING = "warning";
+var STATUS_DANGER = "danger";
+
```

Comment:
Instead of individual variables, use an enum instead.
There can also be a function to get the class for each enum so that you do not have to use a bunch of if-else statements in setStatusMessage.